### ISSUE TITLE
Bootstrap iPod Style jQuery Drill Down Menu Plugin

### ISSUE THREAD

**Issue by mewsoft** [twbs/bootstrap#16178]
Among the built in plugins that I suggest are drill down menu plugin, for my own products needs, so I modified one of jQuery plugins to be exactly what I need "Bootstrap iPod Style jQuery Drill Down Menu Plugin", It has many good things and working well specially for long trees like Classifieds or Auctions categories and with option to display collapsed as a breadcrumb on final node selection or just as a selected menu item by setting the option:

```
show_end_nodes     : true, // drill to final empty nodes, true/false
```
I do not consider my self a JS developer but struggle to do what I want when I do not find it ready online, so I hope some one to re write plugin similar to this one in more compact and stylish code.

Bootply live example  http://www.bootply.com/hIuKeY1K4t

project on github https://github.com/mewsoft/Bootstrap-DrillDownMenu

Sceen shot 
![bootstrap drilldown](https://cloud.githubusercontent.com/assets/7512085/6888119/0a35ecde-d674-11e4-8486-184bf28320bf.png)

**Comment by twbs-lmvtfy** [comment]
Hi @mewsoft!

You appear to have posted a live example (http://s.bootply.com/render/hIuKeY1K4t), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 9, column 9: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.4 ; saw what appears to be usage of Bootstrap 3.3.2
* line 431, column 9: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.4 ; saw what appears to be usage of Bootstrap 3.3.2

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by mewsoft** [comment]
Bootply latest bootstrap version is 3.3.2, so how to fix this to 3.3.4?

**Comment by mdo** [comment]
Not to worry, that's not a huge problem :).

If you're proposing we add a new feature to Bootstrap to do this, we'll be passing on that. That looks like a decently complicated feature, and beyond us not adding anything new to v3, it'd be a bit of work for us to implement in a clear and extensible way. This kind of component is better served as a separate repo, something folks can include in addition to Bootstrap.

Thanks though!
